
---
float64 user_shoe_size
